iron-iconset-svg.html

```diff
@@ -69,6 +69,10 @@
       }
 
     },
+    
+    attached: function() {
+      this.style.display = 'none';
```

Comment:
@cdata @samccone Asked @sorvell; he said `style.display` is better than `hostAttributes`, and it can go either in `ready` or `attached` and it doesn't matter. :balloon: